### ISSUE TITLE
Update to AdminLTE 3

### ISSUE THREAD

**Issue by amontrope** [maiconpinto/cakephp-adminlte-theme#106]
Are there any plans to use AdminLTE 3.x along Cakephp 4.x?

**Comment by Tomicapo** [comment]
Has anyone done this already?

**Comment by maiconpinto** [comment]
@amontrope yes I have plan to use AdminLTE 3.x along CakePHP 4.x. Most likely on CakePHP AdminLTE Theme 3.x.

**Comment by matscav** [comment]
Hi, is there any work on this?

**Comment by QueryTec** [comment]
Hi, are there any news? I saw the closed Pull-Request from amontrope. Can I support you to create the new release? Thx

**Comment by amontrope** [comment]
The pull request was a mistake on my part but I do still have the fork with my own changes. I won't object to support
